libyul/backends/evm/EVMDialect.cpp

```diff
@@ -140,18 +140,16 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 				_assembly.appendAssemblySize();
 			else
 			{
-				yulAssert(
-					_context.subIDs.count(dataName) != 0,
-					"Could not find assembly object <" + dataName.str() + ">."
-				);
-				_assembly.appendDataSize(_context.subIDs.at(dataName));
+				optional<AbstractAssembly::SubID> subId = _context.findSubIDForDataName(dataName);
+				yulAssert(subId,"Could not find assembly object <" + dataName.str() + ">.");
```

Comment:
https://github.com/ethereum/solidity/pull/9047#discussion_r431827298

```suggestion
				yulAssert(subId, "Could not find assembly object <" + dataName.str() + ">.");
```